Documentation/ApiOverview/CommandControllers/Index.rst

```diff
@@ -173,30 +215,11 @@ The above example can be run via command line:
 
    vendor/bin/typo3 yourext:dothings
 
-
 Show help for the command:
 
 .. code-block:: bash
 
    vendor/bin/typo3 help yourext:dothings
 
-.. tip::
-
-   If you installed TYPO3 without Composer, the path for the executable
-   is :file:`typo3/sysext/core/bin/typo3`.
-
-
-.. _symfony-console-commands-scheduler:
-
-Running the Command From the Scheduler
```

Comment:
Why did you remove this completely?

Reply:
Because this is already mentioned above on line 168:
> By default, the command can be used in the scheduler too.